src/immer.ts

```diff
@@ -14,7 +14,7 @@ import {produceEs5} from "./es5"
  * @param {Function} producer - function that receives a proxy of the base state as first argument and which can be freely modified
  * @returns {any} a new state, or the base state if nothing was modified
  */
-export default function produce(baseState, producer) {
+export default function produce(baseState, producer?) {
```

Comment:
Should these parameters have types defined?

Reply:
Eventually yes. I think the reason I added the question mark was to try to make `__tests__/flow/ts.ts` happier

Reply:
The question mark is correct though, produce can be invoked with one argument, basically there are three forms:

```
produce(baseState, producer) => nextState
produce(producer) => baseState => nextState // curried form
produce(producer, initialState) => baseState => nextState // curried form with default state